scripts/attempt-backport.js

```diff
@@ -27,9 +32,9 @@ module.exports = function (app) {
     const options = { owner, repo, prId, logger: event.logger }
 
     debug(`/${owner}/${repo}/pull/${prId} sync`)
-    queueAttemptBackport(options, 7, false)
-    queueAttemptBackport(options, 6, true)
-    queueAttemptBackport(options, 4, true)
+    nodeVersions.forEach(function(node) {
+      queueAttemptBackport(options, node.version, !!node.lts)
```

Comment:
I don't think that's possible since locally scoped arguments are being used inside.